client/packages/android/app/src/main/java/org/openmsupply/client/NativeApi.java

```diff
@@ -513,7 +518,8 @@ public FrontEndHost(JSObject data) {
          * e.g. https://127.0.0.1:8000
          */
         public String getUrl() {
-            return data.getString("protocol") + "://" + data.getString("ip") + ":" + data.getString("port");
+            String host = data.getBool("isLocal") ? "localhost" : data.getString("ip");
```

Comment:
turns out, this isn't strictly needed: if running as server then the client always connects to `127.0.0.1` as the server IP. It accounts for the case where the client connects to the local server from the server discovery screen though, so is worth having.